Readme.md

```diff
@@ -144,6 +144,18 @@ $ rm -rf $CHAIN/vendor/github.com/kr/pretty/.git
 It can change the behavior of cp and put the files
 in the wrong place.)
 
+Since this is a publicy-available open source project, there are some copyright
```

Comment:
sp: public_l_y

> Since this is a publicy-available

Is this the reason for excluding GPL-family licenses? Maybe better to just list the rules without even a token justification?